_posts/2023-06-26-trino-fest-2023-salesforce.md

```diff
@@ -0,0 +1,57 @@
+---
+layout: post
+title: "Trino Fest 2023 recap: Anomaly detection for Salesforce’s production data using Trino"
```

Comment:
Wdty .. do we really need to preface all the blog posts with "Trino Fest 2023 recap:" .. I think we can leave it out.